package.json

```diff
@@ -1,24 +1,31 @@
 {
 	"name": "@s1seven/node-red-generate-pdf",
-	"version": "0.0.5",
+	"version": "0.0.6",
 	"description": "A simple node-red module that wraps the S1Seven generatePdf module and allows it to be used in node-red.",
 	"main": "generate-pdf-node.js",
 	"scripts": {
-		"test": "mocha \"test/**/*_spec.js\""
+		"test": "jest"
 	},
 	"node-red": {
 		"nodes": {
 			"generate pdf": "generate-pdf-node.js"
 		}
 	},
 	"dependencies": {
-		"@s1seven/schema-tools-generate-pdf": "^0.1.1",
+		"@s1seven/schema-tools-generate-pdf": "^0.2.3",
 		"lato-font": "^3.0.0",
 		"lodash": "^4.17.21"
```

Comment:
do you really need lodash ?
Also can reorder the keys so that dependencies comes right before devDependencies ?

Reply:
I don't use `lodash` explicitly, but for some reason `@s1seven/schema-tools-generate-pdf` doesn't resolve the dependency. I'll see if I can find a solution.

As for ordering the keys, done 👍🏼